apps/backup_and_restore/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.9-slim-buster
```

Comment:
I think you want to use debian-old (bullseye) instead of debian-ancient (buster)

Reply:
Fair point, this was just copy/paste from another build. I'll see if I can just upgrade to whatever is the latest.